pkg/controller/jobframework/constants.go

```diff
@@ -32,4 +32,10 @@ const (
 	// ignores this Job from admission, and takes control of its suspension
 	// status based on the admission status of the parent workload.
 	ParentWorkloadAnnotation = "kueue.x-k8s.io/parent-workload"
+
+	// OriginalNodeSelectorsAnnotation is the annotation in which the original
+	// node selectors are recorded upon a workload admission. This information,
+	// if present, will be used to restore them the job is suspended.
```

Comment:
```suggestion
	// if present, will be used to restore them when the job is suspended.
```
Also, we could drop `if present` if there is no scenario the the annotation is empty.

Reply:
done